tests/shopify_python/test_google_styleguide.py

```diff
@@ -225,3 +225,16 @@ def test_use_cond_exprs(self):
         # root_not_msg should add no messages
         with self.assertNoMessages():
             self.walk(root_not_msg)
+
+    def test_lambda_func(self):
```

Comment:
Can you also test a unary operator?
Also add a test that you don't reject `lambda a, b: a * b * c`